### DISCUSSION TITLE
Cant Install Emby LXC

### DISCUSSION THREAD

**Discussion by credge** [tteck/Proxmox#2588] [Help Needed] [answered]
I am trying to pull the Emby LXC using the copy script on the main page and I get to the point where it says "installing Emby" and it bombs out saying..

"- Installing Emby  \
[ERROR] in line 36: exit code 0: while executing command wget -q https://github.com/MediaBrowser/Emby.Releases/releases/download/${LATEST}/emby-server-deb_${LATEST}_amd64.deb"

Any pointers to what I may be doing wrong? I do not have it installed to perform an in-app update..

**Comment by tteck** [comment, marked as answer]
Fixed. With their latest release, they didn't include an `emby-server-deb_4.8.3.0_amd64.deb` file, so I have temporarily set a static version of 4.8.1.0, which included an `emby-server-deb_4.8.1.0_amd64.deb` file.

**Reply by credge** [reply]
Worked a charm! Thanks for all the hard work, and for being super responsive!!

**Reply by ghost** [reply]
Should be okay now for 4.8.3.0 : https://emby.media/linux-server.html
